pom.xml

```
@@ -68,7 +68,7 @@
         <dependency>
             <groupId>com.microsoft.sqlserver</groupId>
             <artifactId>mssql-jdbc</artifactId>
-            <version>12.4.2.jre8</version>
+            <version>12.6.1.jre11</version>
```

```suggestion
            <version>12.4.2.jre8</version>
```

And add an exclusion for the versions plugin: https://github.com/exasol/project-keeper/blob/main/doc/user_guide/user_guide.md#exclude-dependencies-from-automatic-version-update